deps/fdasrsf/optimum_reparamN.pyx

```diff
@@ -0,0 +1,271 @@
+# cython: language_level=2
```

Comment:
Why are we using Python 2 here? Maybe this is why the output of numpy was the traditional one, as new versions have dropped Python 2 support.

Reply:
Changed to python3 in de19847.